api/observations.py

```diff
@@ -144,14 +144,14 @@ def get_coverage_for_station(station, parameters, start_datetime, end_datetime)
             if start_datetime <= time <= end_datetime:
                 values.append(value)
 
-        ranges[p] = NdArray(
+        ranges[p] = NdArrayFloat(
             axisNames=["t", "y", "x"],
             shape=[len(values), 1, 1],
             values=values,
         )
 
     # Add station code
-    station_code = {"inspiregloss:Identifier": station.wsi}
+    station_code = {"eumetnet:locationId": station.wsi}
```

Comment:
This is inconsistent with the locations endpoints, which use the WMO id. 

I would prefer to change the locations endpoint to also use WSI. Leading by example :)